packages/website/src/components/ThemeSection.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import {
+  theme,
+  Heading as FaencyHeading,
+  // AspectRatio,
```

Comment:
In this file, there are some comments as well, want to save them for some reason?